.github/workflows/update-charts-dispatch.yaml

```diff
@@ -61,7 +61,6 @@ jobs:
           helm dependency update charts/testkube-enterprise
 
       - name: Sync & Update Testkube Enterprise Helm chart
-        if: github.event.action != 'trigger-workflow-testkube-agent-main'
```

Comment:
we have github.event.action == 'trigger-workflow-testkube-agent-main' in the step above. This step will also execute on agent update, will it then bump enterprise chart twice?